docs/docs/creating-and-modifying-pages.md

```diff
@@ -8,7 +8,7 @@ Pages can be created in three ways:
 
 - In your site's gatsby-node.js by implementing the API
   [`createPages`](/docs/node-apis/#createPages)
-- Gatsby core automatically turns React components in `src/pages` into pages
+- Gatsby core automatically turns React components in `src/pages/*` into pages. Note that you must make the component the default export
```

Comment:
Some folks reading this page might not know what default exports are exactly, so we shouldn't assume. Linking to more information would help.
```suggestion
- Gatsby core automatically turns React components in `src/pages/*` into pages. Note that you must make the component the [default export](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export)
```